src/sql-parser/src/ast/defs/name.rs

```diff
@@ -60,6 +60,11 @@ impl Ident {
             return Err(IdentError::TooLong(s));
         }
 
+        // Add a condition to check if the name is '.' or '..'
+        if s == "." || s == ".." {
+            return Err(IdentError::InvalidName(s));
+        }
```

Comment:
Now that you have this condition here, you don't need the separate check in webhook planning!

Reply:
Yep, I was going to see the Name changes pass tests then go back and strip out the previous stuff and update the docs etc.